Core/MIPS/ARM64/Arm64RegCache.cpp

```diff
@@ -498,7 +528,11 @@ void Arm64RegCache::FlushArmReg(ARM64Reg r) {
 		mreg.reg = INVALID_REG;
 	} else {
 		if (mreg.loc == ML_IMM || ar[r].isDirty) {
-			_assert_msg_(JIT, mreg.loc != ML_ARMREG_AS_PTR, "Cannot flush reg as pointer");
+			if (mreg.loc == ML_ARMREG_AS_PTR) {
+				// Unpointerify, in case dirty.
```

Comment:
If you originally AND-ed the address with 0x3FFFFFFF, unpointerification by subtraction isn't safe as the high two bits of the address will have been lost. While it may seem unlikely that games would care, pointer subtraction to find out the length of stuff is not uncommon, and if one has lost the top bits and one hasn't.....

Reply:
That's why I removed the ANDing.  Did I miss anywhere?

-[Unknown]

Reply:
Well, I saw that you still used masking here and there (commented)...